style.css

```diff
@@ -0,0 +1,308 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: grid;
+  grid-template:
+    ".   left__menu   main__content     right__menu   .  " 1000px /
```

Comment:
the right__menu grid area is defined, but it looks like no element has been assigned to it - I'm guessing this is probably why your right menu is being display the empty container to the very left of the screen.

Reply:
exactly i will fix still currently working on it thanks